add enabledWorlds config option

WakkaFlocka239 opened the pull request.

bergerkiller commented:
Wait, but `disabledWorlds` already exists. Won't this be super confusing? What if the list is empty, is it then enabled on all worlds? Or disabled on all worlds?

bergerkiller commented on src/main/java/com/bergerkiller/bukkit/tc/TCConfig.java:
```diff
@@ -341,7 +342,18 @@ public static void load(FileConfiguration config) {
+        config.addHeader("enabledWorlds", "World names are not case-sensitive");
+        enabledWorlds.clear();
+        if (!config.contains("enabledWorlds")) {
+            config.set("enabledWorlds", Arrays.asList("DefaultWorld1", "DefaultWorld2"));
```
Dont set to DefaultWorld1/DefaultWorld2, set to Collections.emptyList() instead.

bergerkiller commented on src/main/java/com/bergerkiller/bukkit/tc/TCConfig.java:
```diff
@@ -341,7 +342,18 @@ public static void load(FileConfiguration config) {
         activatorEjectEnabled = config.get("activatorEjectEnabled", true);
 
+        config.setHeader("enabledWorlds", "\nA list of world names where TrainCarts should be enabled");
+        config.addHeader("enabledWorlds", "World names are not case-sensitive");
```
Needs a comment explaining that an empty list means TrainCarts is enabled on all worlds of the server.

WakkaFlocka239 commented:
> Wait, but `disabledWorlds` already exists. Won't this be super confusing? What if the list is empty, is it then enabled on all worlds? Or disabled on all worlds?

I added this because I would much rather do
<img src=https://i.imgur.com/BaEe87k.png>

instead of this
<img src=https://i.imgur.com/PN6y3tk.png>


I've addressed your comments, lmk if there's anything I missed

bbayu123 commented:
I honestly think using a flag to change the list would be more appropriate, for example, a flag to change the blacklist into a whitelist. Up to @bergerkiller to decide though.
